<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="lblHeader">
       <property name="minimumSize">
        <size>
         <width>190</width>
         <height>70</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="btnEditCat">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>108</height>
      </size>
     </property>
     <property name="title">
      <string>View Category List:</string>
     </property>
     <widget class="QPushButton" name="btnIncomeCat">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Income Category</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExpenseCat">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Expense Category</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="btnDeleteCat">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btnAddCat">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="4" colspan="3">
    <widget class="QTableView" name="tblView">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>120</red>
           <green>120</green>
           <blue>120</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>37</number>
     </attribute>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>80</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
